src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/ServiceExtensions.cs

```diff
@@ -30,7 +30,10 @@ public static OrchardCoreBuilder AddOrchardCms(this IServiceCollection services)
 
                 .AddTheming()
                 .AddLiquidViews()
-                .AddCaching();
+                .AddCaching()
+
+                .AddDatabaseShellsConfiguration()
+                ;
```

Comment:
spacing

Reply:
The `AddXxx()` things are already separated by groups

Here the new line is just to start a new group

And the `;` in a new line is just to make easier to add / remove an `AddXxx()`

If this is what you meant ;)